php/lang/security/injection/echoed-request.php

```diff
@@ -44,6 +44,22 @@ function doSmth5() {
     echo "Hello ".trim($_POST['name']);
 }
 
+function doSmth6() {
+   $VAR = $_GET['someval']
+    if(isset($VAR)){ 
+        // ruleid: echoed-request
+        echo $VAR; 
+    }
+}
+
+function doSmth7() {
+    $VAR = $_GET['someval']
```

Comment:
typo, missing semicolon
```suggestion
    $VAR = $_GET['someval'];
```